examples/BUILD.bazel

```diff
@@ -145,7 +145,7 @@ java_fuzz_target_test(
         "JAVA_OPTS": "-Dfoo=not_foo -Djava_opts=1",
     },
     fuzzer_args = [
-        "-fork=5",
+        "-fork=3",
```

Comment:
This change is generally fine if it's just that the CI machines do not have enough RAM to sustain that many forks. But I would like to know by how much this PR increases the peak JVM Heap usage. Do you have any data on that?

Reply:
That was only a test to see if the JVMs would produce OOMs even if the overall memory is sufficient. GitHub Action runners have 7GB memory, which should cause problems when starting 5 JVM with `-Xmx1800m` plus libfuzzer.

Still, the used memory should not increase this dramatically. I finally found the problem, at least in my local tests. ClassGraph uses multiple threads for scanning, reducing those to one should reduce the used memory back to the old amount. Running the failing test locally requires ~130MB heap memory without and ~260MB with ClassGraph with default settings. Using only one scan thread reduces it back to ~130MB. I didn't investigate why the number of scan threads makes such a huge difference.

Reply:
I analyzed heap dumps generated at the end of the test and found that the already closed `ScanResult` still holds a reference to a quite large map containing class name to `ClassInfo` relations. Explicitly removing the reference to the `ScanResult` (what should have been done to begin with) enables GCing it and aligns the heap dumps of both versions.

Reply:
As discussed, there should be no memory increase after startup and the machine is simply overcommitted. Reduced the fork count to `3`.